.github/workflows/validate.yml

```diff
@@ -83,3 +83,10 @@ jobs:
           flags: tests
           name: codecov-umbrella
           verbose: true
+
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
```

Comment:
We include black in our pyproject.toml
https://github.com/aerleon/aerleon/blob/main/pyproject.toml#LL38C21-L38C21
Would using `@stable` here and `^22.8.0` in the toml file potentially cause format flip flops in the future?

Reply:
Installing black through poetry as on line 64 might be a quick way to keep things consistent:
```
      - name: Install library
        run: poetry install --no-interaction
```

Reply:
`isort` should be added as a developer dependency in `pyproject.toml` so that poetry will install both dependencies.

https://github.com/aerleon/aerleon/blob/main/pyproject.toml#L38-L53

Reply:
I don't do a lot of python programming these days.  I am not familiar with poetry.  Let me look into it and perhaps we just run from poetry instead of the black action?

Reply:
No worries and understandable. Looks like Jason was able to fix it.